etc/lf.nu

```diff
@@ -0,0 +1,25 @@
+# Autocompletion for nushell.
+# 
+# Documentation: https://www.nushell.sh/book/externs.html
+
+# Either add this to your existing completions or copy this to
+# your nu config file (Use 'config nu' in the nushell to open it).
+
+module completions {
+ export extern "lf" [
+  --command                                     # command to execute on client initialization
```

Comment:
It might be good to specify the types for these options, for example `--command: string`.  That way `nushell` will provide more helpful error message in case an argument is not provided.